src/core/etl/src/Flow/ETL/Row/Entry/Type/Uuid.php

```diff
@@ -5,7 +5,7 @@
 use Flow\ETL\Exception\InvalidArgumentException;
 use Flow\ETL\Exception\RuntimeException;
 
-final class Uuid implements \Stringable
+final class Uuid implements \Stringable, Type
```

Comment:
hmm that is an interesting approach, I thought this was going to be handled by ObjectType when used in Maps/Lists?

Reply:
Reverted.